index.js

```diff
@@ -1,3 +1,146 @@
+    featuredImage: './images/popimage.png',
+    technologies: ['Html', 'Bootstrap', 'Ruby'],
+    linkToLiveVersion: 'https://summyalena.github.io/Website-Portfolio/',
+    LinkToSource: 'https://github.com/summyalena/Website-Portfolio',
+  },
+  card5: {
+    id: 5,
+    title: 'Professional Art Printing Data',
+    description: "A daily selection of privately personalized reads no accounts or sign-ups required. has been the industry standard Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+    featuredImage: './images/popimage.png',
+    technologies: ['Html', 'Bootstrap', 'Ruby'],
+    linkToLiveVersion: 'https://summyalena.github.io/Website-Portfolio/',
+    LinkToSource: 'https://github.com/summyalena/Website-Portfolio',
+  },
+  card6: {
+    id: 6,
+    title: 'Professional Art Printing Data',
+    description: "A daily selection of privately personalized reads no accounts or sign-ups required. has been the industry standard Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+    featuredImage: './images/popimage.png',
+    technologies: ['Html', 'Bootstrap', 'Ruby'],
+    linkToLiveVersion: 'https://summyalena.github.io/Website-Portfolio/',
+    LinkToSource: 'https://github.com/summyalena/Website-Portfolio',
+  },
+};
+
+const cardSection = document.querySelector('.cardworks');
+
+Object.keys(cardWorks).forEach((cards) => {
+  if (Object.hasOwnProperty.call(cardWorks, cards)) {
+    const card = cardWorks[cards];
+    cardSection.innerHTML += `
+  <div class="card-images">
+    <div class="cardwork-container">
+      <div class=" cardwork cardwork1">
+        <div class="cardcon">
+          <h3>${card.title}</h3>
+          <p>
+          ${card.description.length > 90 ? card.description.substring(0, 90) : card.description}
+          </p>
+          <div class="spancon">
+            <span class="span">HTML</span>
+            <span class="span">CSS</span>
+            <span class="span">JAVASCRIPT</span>
+          </div>
+        </div>
+        <div class="button">
+          <a href="" id= "${card.id}" class="see" >See Project</a>
+        </div>
+      </div>
+    </div>
+  </div>`;
+  }
+});
+
+const projectPopup = document.querySelector('.project-popup');
+const popButton = document.querySelectorAll('.see');
+
+popButton.forEach((popup) => {
```

Comment:
- [ ] [OPTIONAL] Everything is good 👏. However, it would be better if you add the popup and the click event to this first project card which is bigger than the remaining six. When I clicked on the `see project` button, I was not able to see the popup. Look at the screenshot below to see the project card I am referring to. 👇 
![image](https://user-images.githubusercontent.com/90222110/180315557-2984b0b1-1a06-4642-aa8d-f0356c7bbcf8.png)